### PR TITLE
making repo python3 compatible

### PR DESCRIPTION
**Pull request by kedarisetti**

### CODE REVIEW

**Comment by amarghosh** [comment]
Thanks for the PR. As you can imagine, I haven't used this in a while. Now that we are long past the sunset of python2, is there any point in keeping `sys.version_info` based support for that version anymore? Why not replace it with just the python3 code? And based on a quick look, some old style print statements are still present in the code. It would be great if you can handle them as well.

**Comment by kedarisetti** [comment]
@amarghosh made changes as mentioned in your comment.

File: src/gui.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```
Comment (qarkai) [inline src/gui.py:1]: Shebang is unused.

**Comment by qarkai** [comment]
@amarghosh Any feedback?

**Comment by amarghosh** [comment]
Thanks for the PR and the comments, and apologies for the delay in response. I took a quick look at the changes and it looks fine, and ran it against few mp4 files and it seems to work fine. I am merging it in.

**Pull request merged**